core/main/src/Utils/CanvasUtils.ts

```diff
@@ -460,4 +460,33 @@ export class CanvasUtils {
             context.restore();
         }
     }
+
+    public static drawEllipse(
+        context: CanvasRenderingContext2D,
+        particle: IParticle,
+        fillColorValue: IHsl | undefined,
+        radius: number,
+        opacity: number,
+        width: number,
+        rotation: number
+    ): void {
+        const pos = particle.getPosition();
+        context.beginPath();
+
+        if (fillColorValue) {
+            context.strokeStyle = ColorUtils.getStyleFromHsl(fillColorValue, opacity);
+        }
+
+        if (width == 0) {
+            return;
+        } else {
```

Comment:
Codacy found an issue: [The preceding `if` block ends with a `return` statement. This `else` is unnecessary.](https://app.codacy.com/gh/matteobruni/tsparticles/pullRequest?prid=6406861)